### DISCUSSION TITLE
Github student copilot

### DISCUSSION THREAD

**Discussion by sanadsa** [community/community#139702] [GitHub Education]
### Hi everyone,

I have been approved before 5 days for github student but I still dont have access to copilot.
Thanks

**Comment by davevad93** [comment]
Hi @sanadsa ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sanadsa was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by enesdoruk** [comment]
How to get the student copilot?

**Reply by davevad93** [reply]
Hey @enesdoruk , to get Copilot for free you have to apply for the `Student Developer Pack` [here](https://education.github.com/discount_requests/application?type=student). Make sure you meet all the requirements to apply successfully:

- [Apply to GitHub Education as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student)

**Reply by kaavyabaranwal** [reply]
> Hey @enesdoruk , to get Copilot for free you have to apply for the `Student Developer Pack` [here](https://education.github.com/discount_requests/application?type=student). Make sure you meet all the requirements to apply successfully:
> 
> * [Apply to GitHub Education as a student](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student)

@davevad93 Does it also get re-activated if I renew my membership?

**Reply by brenoepics** [reply]
@kaavyabaranwal  did your github copilot/student also suddenly expire a few hours ago? now I have github pro back but no copilot.

**Reply by kaavyabaranwal** [reply]
> @kaavyabaranwal did your github copilot/student also suddenly expire a few hours ago? now I have github pro back but no copilot.

@brenoepics no, my copilot expired normally (in one year) . someone told me to renew my student membership and it should work.
